wgpu-core/src/device/any_device.rs

```diff
@@ -34,7 +34,7 @@ impl AnyDevice {
         unsafe fn drop_glue<A: HalApi>(ptr: *mut ()) {
             // Drop the arc this instance is holding.
             unsafe {
-                _ = Arc::from_raw(ptr.cast::<A::Surface>());
+                _ = Arc::from_raw(ptr.cast::<A::Device>());
```

Comment:
This looks like a copy-paste error introduced in https://github.com/gfx-rs/wgpu/pull/5081 (CC @udoprog, the author, as an FYI, no action items I can see).